deploy_polygon/06_giveaway/01_setup_multi_giveaway_1.ts

```diff
@@ -48,6 +63,16 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
       ozdRelayer
     );
   }
+  if (!multiGiveawayAdminIsAdmin) {
+    await execute(
+      'PolygonMulti_Giveaway_1',
+      {from: deployerIsAdmin ? deployer : multiGiveawayAdmin, log: true},
```

Comment:
should be .. sandAdmin not multiGiveawayAdmin?

Reply:
We have sandAdmin as DEFAULT_ADMIN_ROLE and multiGiveawayAdmin as MULTI_GIVEAWAY_ROLE

sandAdmin is set on line 54 already